sdk/src/main/java/io/dapr/actors/runtime/DaprClientBuilder.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.actors.runtime;
+
+import okhttp3.OkHttpClient;
+import io.dapr.actors.*;
+
+/**
+ * Builds an instance of DaprAsyncClient or DaprClient.
+ */
+public class DaprClientBuilder {
```

Comment:
Why public since this is only for the runtime?

Reply:
Removed public.